schema/project-level/codelists/climateMeasures.csv

```diff
@@ -0,0 +1,10 @@
+Code,Title,Description,Source
+embodiedEnergyReduction,Reduction of embodied energy,"Measures that reduce the sum of energy required to produce, transport and repair or replace the materials used by the project.",OC4IDS
+embodiedEmissionReduction,Reduction of embodied emissions,"Measures that reduce the sum of emissions generated by the production, transportation and replacements or repair of materials used by the project.",OC4IDS
+fugitiveEmissionReduction,Reduction of fugitive emissions,"Measures that reduce greenhouse gas emissions and air pollutants from intentional or unintentional releases, such as methane during transport of natural gas or HFC emission from refrigeration equipment.",OC4IDS
+renewableEnergySources,Use of renewable energy sources,The project adopts renewable energy sources and minimizes the use of fossil fuels as an energy source.,OC4IDS
+carbonCapture,Use of carbon capture technology,The project incorporates carbon capture and storage technologies.,OC4IDS
+recycledMaterials,Use of recycled and/or reused materials,"Measures that reduce the use of virgin natural resources and avoid sending useful materials to landfills by specifying reused materials, including structures, and material with recycled content.",OC4IDS
+regenerativeDesign,Regenerative design,"The project has used a regenerative design approach. Regenerative design is a system of technologies and strategies, based on an understanding of the inner working of ecosystems that generates designs to regenerate rather than deplete underlying life support systems and resources within socio-ecological wholes.",OC4IDS
+locationAnalysis,Location analysis,An assessment of the project location has been undertaken to avoid or mitigate against site specific future climate related risks.,OC4IDS
```

Comment:
Align with conservationMeasures.csv

```suggestion
locationAnalysis,Location analysis,Location analysis was performed to select the site with the minimum future climate-related risks.,OC4IDS
```

Reply:
@jpmckinney  In conservationMeasures.csv , **minimising** impact is explicitly mentioned in the origins:  
_"location analysis to pick a site that minimises negative impacts associated with construction and operation including minimising use of green-field land."_ [#401#issuecomment-1781377630](https://github.com/open-contracting/infrastructure/issues/401#issuecomment-1781377630)

In the origins of locationAnalysis in the context of climateMeasures.csv, minimising is not the intent: 
_"The project is designed/sited so as to avoid or mitigate site-related risks. CR2.1"_[ #402#issuecomment-1785755223  ](https://github.com/open-contracting/infrastructure/issues/402#issuecomment-1785755223 )
The usage of **mitigate** need not mean that the location with the minimum climate risk was chosen. 

So while the codes themselves have identical titles, does the following description work?

`Location analysis was performed to avoid or mitigate future climate-related risks.`

Reply:
> Location analysis was performed to select the site, in order to avoid or mitigate future climate-related risks.

?

Reply:
> > Location analysis was performed to select the site, in order to avoid or mitigate future climate-related risks.
> 
> ?

Thanks, I'll make the change.